mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala

```diff
@@ -241,7 +241,9 @@ class CountVectorizer @Since("1.5.0") (@Since("1.5.0") override val uid: String)
     }
     wordCounts.unpersist()
 
-    require(vocab.length > 0, "The vocabulary size should be > 0. Lower minDF as necessary.")
+    if (vocab.isEmpty) {
+      this.logWarning("The vocabulary size is empty. Alter minDF/maxDF as necessary.")
```

Comment:
Minor, but it's `minTF` (it was wrong before). Also we could suggest lowering minTF, or increasing maxDF, more specifically.

Reply:
Shouldn't it be `minDF` instead of `minTF`?

As per [this](https://github.com/apache/spark/blob/f73ca919251f9c054671ddbf3fecb8f005d66836/mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala#L119), minTF is only used in transform and not fitting. `minDF` and `maxDF` are used to filter out words in vocabulary during fitting.

Reply:
Oops I think you're right. Still we could retain the advice to decrease the min or increase the max

Reply:
Yes :). Thanks for the suggestion. That definitely makes it better. Made the change. Is this fine?